Create 2016-11-09-Using-HALBuilder-with-german-characters.md

fiatfilip opened the pull request:
Added a new blog post on how to make sure that umlauts are properly displayed when using HALBuilder

Commit:
Create 2016-11-09-Using-HALBuilder-with-german-characters.md

DonMartin76 commented:
Hey Filip. I have a couple of issues:
- You didn't add yourself to `_authors.yml`, please do that (and make sure the key matches what you have in your post); in case you don't want to link to anything, you could leave it as is as well
- Please add an empty line after each section header (`###`)
- Please check which tags are available in `_tags.yml`; `halbuilder` is not one of them ;-)

The blog as it is right now is very short; would it maybe make sense to write some short generic intro on the HAL builder while you're at it? Is it good?

BR, Martin

Commit:
Update 2016-11-09-Using-HALBuilder-with-german-characters.md

Applied some of the reviewer's suggestions.
Regarding the length of the article... this will take some time ;)

DonMartin76 commented:
> Regarding the length of the article... this will take some time ;)

Is that a "will do" or "will not do"?

Depending on that, I'd merge now, or not.

fiatfilip commented:
Hi Martin, it is a "will do" so please do not merge yet, because after I finish my current "quite urgent" task, I will prepare a more extended article.
Thank you,
Filip

Commit:
Update 2016-11-09-Using-HALBuilder-with-german-characters.md

Hi Martin, 
please validate my changes and give me your feedback.
Thanks,
Filip
